### PR TITLE
Added Helpers examples in expressions.md

### PR DESCRIPTION
**Pull request by eswarijayakumar**
Hello,

I have added Examples for the helpers in expressions.md

### CODE REVIEW

File: src/guide/expressions.md [hunk 1]
```diff
@@ -81,77 +81,88 @@ below.
...
+The output of above helper is generated as below
+
+<ExamplePart examplePage="/examples/helper-hash-arguments.md" show="output" />
 
 Handlebars provides additional metadata, such as Hash arguments, to helpers as a final parameter.
 
 Handlebars also offers a mechanism for invoking a helper with a block of the template. Block helpers can then invoke
 that block zero or more times with any context it chooses.
 
-[Learn More: Block Helpers](block_helpers.html)
```
Comment (nknapp) [inline src/guide/expressions.md]: If you use `!button[Learn More...](block-helpers.html)` you'll get an orange button that looks like the other `Learn More`-links. Could you do that?

**Comment by nknapp** [comment]
Hi and thanks a lot. I have added a comment to one line. I'll just wait a little before merging so you have to do add it, if you like.

**Comment by nknapp** [comment]
I'm merging the request now, because there are other things to do on the expressions-page...

**Comment by nknapp** [comment]
There are some things I would like to change:

* Use the example from "/examples/helper-simple.md" as an example for a helper with a single parameter. 
* Think of another example with multiple parameters

The main reason is that the "Handlebars.SafeString"-function is introduced later in the chapter and should be avoided in the text before.

**Comment by eswarijayakumar** [comment]
Sorry! I was traveling yesterday so I couldn't respond immediately. 

Thanks for the suggestions. Great to see that the suggested changes are implemented already.

I will look into the multiple parameters case and try to make a PR soon.

**Comment by nknapp** [comment]
I thik I changed the order and added the Safestring example first, so it should be OK now

**Pull request merged**